android/services/mongodb-remote-perftests/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/performance/SyncPerformanceTests.kt

```diff
@@ -24,91 +25,89 @@ class SyncPerformanceTests {
         }
     }
 
-    private var testHarness = SyncPerformanceIntTestsHarness()
-
-    @Before
-    fun setup() {
-        this.testHarness.setup()
-    }
-
-    @After
-    fun teardown() {
-        this.testHarness.teardown()
-    }
+    // private var testHarness = SyncPerformanceIntTestsHarness()
```

Comment:
[nit] remove this comment
[q] also why is this out of the class now?

Reply:
This was the reason that the network transport only worked the first time. The references to the transport got very strange. We had to do the same thing in iOS. Regardless it makes sense for us to only have one harness.

Reply:
+1 remove comments